pkg/kv/kvserver/replica_raftstorage.go

```diff
@@ -704,6 +658,60 @@ func (r *Replica) applySnapshot(
 	return nil
 }
 
+func writeUnreplicatedSST(
+	ctx context.Context,
+	id storage.FullReplicaID,
+	st *cluster.Settings,
+	meta raftpb.SnapshotMetadata,
+	hs raftpb.HardState,
+	sl *logstore.StateLoader,
+) (*storage.MemFile, bool, error) {
```

Comment:
Please add a func comment, and/or name the output variables so that the returned `bool` is explained.

Reply:
Done. I also added a TODO to look into this parameter in the first place; won't it always be true? Once we have a datadriven test we'll know for sure.